integration_tests/e2e/induction/createChangeInductionQuestionSet.cy.ts

```diff
@@ -190,31 +208,19 @@ context(`Change new Induction question set by updating 'Hoping to work on releas
       .hasBackLinkTo(`/prisoners/${prisonNumberForPrisonerWithNoInduction}/create-induction/want-to-add-qualifications`)
       .submitPage()
 
-    // 'Has the prisoner worked before' is the next page. This is not asked on the short question set.
+    // 'Has the prisoner worked before' is the next page. This is asked on the short question set, so this will already have an answer set
```

Comment:
We need to make a couple of changes to the cypress test that does the inverse of the previous one - ie.  "creating a Short question set induction; but change it to a Long question set induction via the change link for 'do they want to work on release' on the Check Your Answers page.

The reason this test needs to change is not because they are new screens/questions in the Long question set, but because the Short question set (from which we are changing to a Long question set!) now already has the answers set, which it didnt have before.